src/controllers/chat.controller.js

```diff
@@ -0,0 +1,124 @@
+require('dotenv').config();
+const path = require('path');
+
+const db = require(path.join(__dirname, '../db/index'));
+const queries = require(path.join(__dirname, '../db/queries'));
+const getPayload = require(path.join(__dirname, '../helpers/jwt'));
+
+const addUser = async (userId, chatId) => {
+    try {
+        await db.none(queries.addUserToChat, [userId, chatId]);
+    }
+    catch(error) {
+        return error.constraint === 'users_chats_pkey' ? 
+            Promise.reject('User already member of chat') :
+            Promise.reject(error);
+    }
```

Comment:
This is just to be extra careful that we only specify the user was already added to the chat/is already a member of the chat if the primary key constraint was violated. Otherwise, whatever error due to incorrect values provided in the prepared statement will be thrown instead.